proto/cosmos/base/abci/v1beta1/query.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+package cosmos.base.abci.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "tendermint/crypto/proof.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/types";
+
+// Query exposes the ABCIApplication Query method on BaseApp over gRPC, critical for obtaining proofs over gRPC
+service Service {
+  // Query exposes the ABCIApplication Query method on BaseApp
+  rpc Query(QueryRequest) returns (QueryResponse);
+}
```

Comment:
Hey @ChristianBorst, thanks! We already have a Tendermint proxy service in `proto/cosmos/base/tendermint/v1beta1/query.proto`. What do you think of putting this there?

Reply:
I'm fine with that, and it should resolve @aaronc's request about putting the query endpoint somewhere other than the root of types/.